unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Analytics/BIWAnalytics.cs

```diff
@@ -0,0 +1,201 @@
+    /// </summary>
+    /// <param name="source">It comes from the shortcut or from BuilderPanel</param>
+    public static void StartEditorFlow(string source)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("source", source);
+        AddLandInfoSendEvent("start_editor_flow", events);
+    }
+
+    public static void EnterEditor(float loadingTime)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("loading_time", loadingTime.ToString());
+        AddLandInfoSendEvent("enter_editor", events);
+    }
+
+    public static void ExitEditor(float timeInvestedInTheEditor)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("time_in_the_editor", timeInvestedInTheEditor.ToString());
+        AddLandInfoSendEvent("exit_editor", events);
+    }
+
+    public static void StartScenePublish(SceneMetricsModel sceneLimits)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("scene_limits", ConvertSceneMetricsModelToDictionary(sceneLimits).ToString());
+        AddLandInfoSendEvent("start_publish_of_the_scene", events);
+    }
+
+    public static void EndScenePublish(SceneMetricsModel sceneLimits, string successOrError, float publicationTime)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("success", successOrError);
+        events.Add("publication_time", publicationTime.ToString());
+        events.Add("scene_limits", ConvertSceneMetricsModelToDictionary(sceneLimits).ToString());
+        AddLandInfoSendEvent("end_scene_publish", events);
+    }
+
+    public static void SceneLimitsOverPassed(SceneMetricsModel sceneLimits)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("scene_limits", ConvertSceneMetricsModelToDictionary(sceneLimits).ToString());
+        AddLandInfoSendEvent("scene_limits_over_passed", events);
+    }
+
+    /// <summary>
+    /// When a new item is placed from the catalog
+    /// </summary>
+    /// <param name="catalogItem">The item that has been added</param>
+    /// <param name="source">It has been added from Categories, Asset pack, Favorites or Quick Access</param>
+    public static void NewObjectPlaced(CatalogItem catalogItem, string source)
+    {
+        Dictionary<string, string> events = new Dictionary<string, string>();
+        events.Add("name", catalogItem.name);
+        events.Add("assetPack", catalogItem.assetPackName);
+        events.Add("category", catalogItem.category);
+        events.Add("category Name", catalogItem.categoryName);
+        events.Add("source", source);
+        events.Add("type", catalogItem.itemType.ToString());
+        AddLandInfoSendEvent("new_object_placed", events);
+    }
+
+    public static void QuickAccessAssigned(CatalogItem catalogItem, string source)
```

Comment:
What is this? (It's missing comments and i can't deduce it)

Reply:
In builder in world we have a quick access buttons  ( from 1 to 9) where you can set assets to quickly drop them in the scene, each time a asset is assigned to this part, an event is fired with the item assigned

Reply:
ty <3